app/models/spreadsheet_exporter.rb

```diff
@@ -34,18 +34,19 @@ class MasterInventory < SpreadsheetExporter
     def initialize
       super
       sheet1 = create_sheet("All Items")
-      sheet1 << %w(Category Description Quantity\ On\ hand SKU Value)
+      sheet1 << %w(itemRef Description Quantity\ On\ hand SKU Value)
```

Comment:
The original code had `Category` as the first column and then header rows with the category name in that column with nothing else there... now the category row is still there but fills a column that has multiple purposes... is this change intentional?

As for naming it `itemRef`... this is a generic XLS spreadsheet export rather than a NetSuite specific CSV export... should we just call this `ID` instead, or did Lisa prefer `itemRef`? Personally, I'm not much of a fan of NetSuite terminology existing outside the NetSuite exports.

Should we consider a separate master inventory export that does CSV explicitly for NetSuite?

Reply:
👍